sdk/tests/conformance/textures/misc/copy-tex-sub-image-2d-partial-texture.html

```diff
@@ -43,36 +43,141 @@
 "use strict";
 description("Verifies that copyTexSubImage2D redefining part of the destination texture works as expected.");
 
+////
+
+var kWidth = 16;
+var kHeight = 16;
+
+////
+
+var wtu = WebGLTestUtils;
+var canvas = document.getElementById("canvas");
+
+canvas.width = kWidth;
+canvas.height = kHeight;
+var gl = wtu.create3DContext(canvas);
+
+////
+
+function clearTo(color) {
+    gl.clearColor(color[0], color[1], color[2], color[3]);
+    gl.clear(gl.COLOR_BUFFER_BIT);
+}
+
+function readInto(view) {
+    gl.readPixels(0, 0, kWidth, kHeight, gl.RGBA, gl.UNSIGNED_BYTE,
+                  new Uint8Array(view.buffer));
+}
+
+////
+
 function runTest() {
-    var wtu = WebGLTestUtils;
-    var canvas = document.getElementById("canvas");
-    canvas.width = 256;
-    canvas.height = 256;
```

Comment:
This test (added in #1992 ) specifically allocated a large canvas which would be GPU-accelerated in Chrome, and acted as a regression test for a bug. The new test is great, but it has significantly different behavior. Rather than changing this test, could you please add a new one? Feel free to move this one into something like copy-tex-sub-image-2d-partial-texture-from-large-canvas.html if desired.